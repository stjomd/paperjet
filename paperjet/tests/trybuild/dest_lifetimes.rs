fn main() {
	let _ = {
		let mut destinations = paperjet::unix::dest::CupsDestinations::new();
		destinations.into_iter().next().unwrap()
	};
}
