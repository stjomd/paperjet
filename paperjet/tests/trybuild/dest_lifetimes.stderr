error[E0597]: `destinations` does not live long enough
 --> tests/trybuild/dest_lifetimes.rs:4:3
  |
3 |         let mut destinations = paperjet::unix::dest::CupsDestinations::new();
  |             ---------------- binding `destinations` declared here
4 |         destinations.into_iter().next().unwrap()
  |         ^^^^^^^^^^^^ borrowed value does not live long enough
5 |     };
  |     - `destinations` dropped here while still borrowed
